sda-commons-client-jersey-wiremock-testing/build.gradle

```diff
@@ -7,8 +7,12 @@ dependencies {
      */
     exclude group: 'javax.servlet', module: 'javax.servlet-api'
   }
-
+  compile 'org.slf4j:jcl-over-slf4j'
```

Comment:
Can you double check if this dependency is really needed?

Reply:
WiremockServer needs it for starting, but I moved it to testCompile